### PR TITLE
documentation: ZENKOIO-140_blobserver_operation

### PR DESCRIPTION
**Pull request by wabernat**
Operation Guide for Blobserver.

updated version to 1.2

Added blobserver API access, scanned document looking for Azure Blob &
Blobserver exceptions to S3 & CloudServer statements.

fixes #ZENKOIO-140

### CODE REVIEW

File: docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/index.rst [hunk 1]
```diff
@@ -17,12 +17,12 @@ Orbit is configured, you can unlink it from a Zenko instance as well.
 Registering Orbit to Zenko
 --------------------------
 
-Orbit can be run as a user interface to Zenko no matter where or how
-Zenko is hosted. You can deploy Zenko:
+Orbit can be run as a user interface to Zenko no matter where or how Zenko is
+hosted. You can deploy Zenko on any host running Kubernetes 1.11.3 or
```
Comment (JianqinWang) [inline docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/index.rst:21]: I recall there may have been some issue with 1.11.x? or was that something else @tmacro

Comment (tmacro) [reply docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/index.rst:21]: A couple of the ease-of-use admin features I added require a newer version, but as the helm chart with blobserver hasn't been officially released yet I think we're good with `1.11.3`. The features are more for the sysadmin anyway (and are optional) and don't impact  Zenko functionality.

File: docs/docsource/operation/Zenko_CLI/index.rst [hunk 2]
```diff
@@ -101,7 +100,70 @@ AWS gateway.
 
 Zenko can now respond to the set of S3 commands documented in the
 :version-ref:`Zenko Reference
-<https://documentation.scality.com/Zenko/{version}/reference/index.html>`.
+<https://documentation.scality.com/Zenko/{version}/reference/index.html>`.       
+
+Set Up Blobserver API Access
```
Comment (tmacro) [inline docs/docsource/operation/Zenko_CLI/index.rst:105]: Perhaps a note here that setting up HTTPS is required for using the SRP API with `az`

**Review by tmacro — APPROVED** [review]

**Review by JianqinWang — APPROVED** [review]

**Comment by wabernat** [comment]
@bert-e approve

**Pull request merged**